packages/workflows/src/handlers/cart/attach-customer-details-to-cart.ts

```diff
@@ -0,0 +1,61 @@
+import { validateEmail } from "@medusajs/utils"
+
+import { WorkflowArguments } from "../../helper"
+
+type AttachCustomerDetailsDTO = {
+  customer_id?: string
+  email?: string
+}
+
+type HandlerInputData = {
+  cart: {
```

Comment:
I still think we should scope it, that would make it easier for the handler to accept from other handlers and for each of those to not clash. Maybe not for this, but it would be good to have a generic structure for these handlers.

We could still make it more reusable.
```
{
  customer: {
    customer_id?: string
    email?: string
  }
}
```